.maintain/run-benchmark

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+#### Function definitions
+print_help () {
+  echo "USAGE: \\n\
+  \\trun-benchmark.sh --all\\n\
+  \\trun-benchmark.sh --pallet <pallet-name>\\n\
```

Comment:
Nit: This is not accurate since the filename is 'run-benchmark'. I think I would prefer it to be named 'run-benchmark.sh' anyways, but if we keep the current name we should maybe change this help statement.

Reply:
you are right, I left `run-benchmark`